src/components/navbar/index.js

```diff
@@ -2,42 +2,76 @@ import React from 'react';
 import styled from 'styled-components';
 
 const NavWrapper = styled.header`
-  background-color: lightpink;
-  padding: 10px;
-  color: black;
+  position: fixed;
+  background: #48C6DC;
+  padding: 1rem; 
+  width: 100vw;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font: Roboto;
+  font-style: normal;
 
   nav {
     height: 100%;
   }
 `;
 
+const NavHeader = styled.a`
+  margin-left: 3vw;   
+  margin-top: 2vh;
+  text-decoration: none;
+  color: white;
+  font-size: 1.5rem;
+  font-weight: 500;
+  align-self: flex-start;
+`;
+
 const NavLinksContainer = styled.ul`
   display: inline-block;
-  padding: 0;
-  margin-top: 20px;
 
   li {
     display: inline;
-    margin-right: 30px;
+    margin-right: 40px;
+  }
+
+  a {
+    text-decoration: none;
+    color: white;
+    font-size: 1rem;
+    font-weight: 500;
   }
 `;
 
+const Button = styled.div`
```

Comment:
because this is a button, try using button instead of div here-- I think the styling might be a bit easier too 😄